numba/tests/test_support.py

```diff
@@ -315,6 +316,7 @@ def test_nptimedelta(self):
 
 class TestMisc(TestCase):
 
+    @skip_as_broken_if_freethreading
```

Comment:
FYI: float constants are interned and immortalized in the free threaded build, so if you want something that you can test reference counting with use `object()`:

```python
x = object()
y = object()
...
```